Tests/Unit/NetworkServer/LoRaDeviceTest.cs

```diff
@@ -226,10 +229,8 @@ public async Task When_Initialized_ABP_Device_Should_Have_All_Properties()
         [Theory]
         [InlineData("false")]
         [InlineData("FALSE")]
-        [InlineData("0")]
         [InlineData(0)]
         [InlineData(false)]
-        [InlineData("ture")] // misspelled "true"
```

Comment:
Why remove this case?

Reply:
I think this was wrong. The default is true. When we are getting a value we can't parse and we changed the default? I fixed that to keep the default for values we can't parse. I think that makes sense. Sorry, I wanted to mention that in my notes and forgot about it, but good catch. :)